<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TTextBoxDialog</class>
 <widget class="QDialog" name="TTextBoxDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enter Text ...</string>
  </property>
  <property name="locale">
   <locale language="English" country="AmericanSamoa"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutVBox">
   <item>
    <widget class="QTextEdit" name="textEditText">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelUnicode">
     <property name="text">
      <string>Unicode Send Command Value</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEditUnicode">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Panel</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
     </property>
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string notr="true"/>
     </property>
     <property name="backgroundVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxPreview">
     <property name="text">
      <string>Preview Using Font</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="locale">
      <locale language="English" country="AmericanSamoa"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayoutHBox">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCharacterMap">
        <property name="text">
         <string>Character Map</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonOk">
        <property name="text">
         <string>Ok</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonOk</sender>
   <signal>clicked()</signal>
   <receiver>TTextBoxDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>TTextBoxDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
